test/functional/apps/settings/_advanced_settings.js

```diff
@@ -35,6 +35,12 @@ define(function (require) {
           .then(function (advancedSetting) {
             expect(advancedSetting).to.be('America/Phoenix');
           })
+          .then(function (advancedSetting) {
```

Comment:
If you're not using the advancedSetting, there's no need to declare it as an argument.  Same goes for the function below.